schemas/context/segmentmembership.schema.json

```diff
@@ -29,6 +29,67 @@
           "format": "date-time",
           "description": "The timestamp when the assertion of segment membership was made."
         },
+        "xdm:payload" : {
+          "title" : "Payload",
+          "description": "Values that are directly related with the segment realization. This payload exists with the same validUntil as the segment realization.",
+          "oneOf": [
+            {
+              "type": "object",
+              "properties": {
+                "xdm:payloadNumberValue": {
+                  "title": "Value",
+                  "type": "number",
+                  "description": "The number"
+                },
+                "xdm:payloadType": {
+                  "title": "Payload Type",
+                  "type": "string",
+                  "const": "number",
+                  "description": "The type of payload."
+                }
+              },
+              "required": ["xdm:payloadNumberValue", "xdm:payloadType"]
+            },
+            {
+              "type": "object",
+              "properties": {
+                "xdm:payloadPropensityValue": {
+                  "title": "Value",
+                  "type": "number",
+                  "description": "The propensity",
+                  "minimum": 0,
+                  "maximum": 1
+                },
+                "xdm:payloadType": {
+                  "title": "Payload Type",
+                  "type": "string",
+                  "const": "propensity",
+                  "description": "The type of payload."
+                }
+              },
+              "required": ["xdm:payloadPropensityValue", "xdm:payloadType"]
+            },
+            {
+              "type": "object",
+              "properties": {
+                "xdm:payloadBooleanValue": {
+                  "title": "Value",
+                  "type": "boolean",
+                  "description": "The propensity"
```

Comment:
the description looks like a cut and paste error - description for type boolean should probably be something like "true or false"

Reply:
fixed -- thanks!